akka-docs/src/main/paradox/scala/distributed-pub-sub.md

```diff
@@ -30,9 +30,13 @@ any other node.
 There a two different modes of message delivery, explained in the sections
 [Publish](#distributed-pub-sub-publish) and [Send](#distributed-pub-sub-send) below.
 
+@@@ div { .group-scala }
+
 A more comprehensive sample is available in the
 tutorial named [Akka Clustered PubSub with Scala!](https://github.com/typesafehub/activator-akka-clustering).
 
```

Comment:
As activator is deprecated indeed I'd say we don't want to link to that one anymore, but migrate any interesting samples to tech hub. In this case I think we can link to http://dev.lightbend.com/start/?group=akka&project=akka-samples-cluster-scala / http://dev.lightbend.com/start/?group=akka&project=akka-samples-cluster-java